packages/components/organisms/f-checkout/src/components/_tests/helpers/setup.js

```diff
@@ -74,13 +93,23 @@ const i18n = {
     }
 };
 
-const createStore = (state = defaultState, actions = defaultActions) => new Vuex.Store({
+const createStore = (
+    state = defaultState,
+    actions = defaultActions,
```

Comment:
We should probably rename these to `checkoutState = defaultCheckoutState` and `checkoutActions = defaultCheckoutActions`.

Reply:
I've updated all of these now 👍